src/clr-angular/utils/_helpers.clarity.scss

```diff
@@ -63,6 +63,12 @@ $opp-directions: (
   }
 }
 
+@mixin fixForIE9AndUp() {
```

Comment:
I would rather not call this fixForIE9AndUp because Clarity only supports IE11 and up. Yes, I know there is one below with the name fixForIE10AndUp but that is from when we used to support IE10 and it should probably be removed.

What do you think about fixForIE11 - since this is only for IE11.

Reply:
i tried to use the fixForIE11 but it didnt work. also, i see that that function its not used anywhere. maybe we should find another hack that works only for IE11. what do you think?

Reply:
I think this is the right approach. What happened that prevented `fixForIE11` from working? Was there an error you saw? Could it have anything to do with the numbers at the end?

Reply:
Lets try the recommendation below from @gnomeontherun before we proceed with the hacky solution.